### ISSUE TITLE
Cannot tap mattprecious/repo: invalid syntax in tap

### ISSUE THREAD

**Issue by oldergod** [mattprecious/protogram#51]
```
$ brew install mattprecious/repo/protogram

Running `brew update --preinstall`...
==> Auto-updated Homebrew!
Updated 3 taps (homebrew/core, homebrew/cask and homebrew/services).
==> New Formulae
sophus
==> Updated Formulae
Updated 73 formulae.
==> New Casks
descript                                                                                                                                                                               manymc
==> Updated Casks
Updated 37 casks.

==> Tapping mattprecious/repo
Cloning into '/opt/homebrew/Library/Taps/mattprecious/homebrew-repo'...
remote: Enumerating objects: 8, done.
remote: Counting objects: 100% (8/8), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 8 (delta 1), reused 8 (delta 1), pack-reused 0
Receiving objects: 100% (8/8), done.
Resolving deltas: 100% (1/1), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/mattprecious/homebrew-repo/Formula/protogram.rb
protogram: Calling bottle :unneeded is disabled! There is no replacement.
Please report this issue to the mattprecious/repo tap (not Homebrew/brew or Homebrew/core):
  /opt/homebrew/Library/Taps/mattprecious/homebrew-repo/Formula/protogram.rb:8

Error: Cannot tap mattprecious/repo: invalid syntax in tap!
```

**Comment by oldergod** [comment]
I'm seeing another one

```
Cloning into '/opt/homebrew/Library/Taps/mattprecious/homebrew-repo'...
remote: Enumerating objects: 13, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 13 (delta 2), reused 8 (delta 1), pack-reused 0
Receiving objects: 100% (13/13), done.
Resolving deltas: 100% (2/2), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/mattprecious/homebrew-repo/Formula/protogram.rb
protogram: Unsupported special dependency :java
Error: Cannot tap mattprecious/repo: invalid syntax in tap!
```

**Comment by mattprecious** [comment]
Should be fixed now with mattprecious/homebrew-repo#4 and mattprecious/homebrew-repo@b016da00a62d425475da0c60474dd4f92730f8bd

**Comment by oldergod** [comment]
Confirmed it's fixed! Thank you
